extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.js

```diff
@@ -130,8 +131,24 @@ export class AmpStorySubscriptions extends AMP.BaseElement {
       'amp-subscriptions-dialog'
     );
     ampSubscriptionsEl.addEventListener('click', (event) =>
-      this.onSkipButtonClick_(event)
+      this.onSubscriptionsDialogClick_(event)
     );
+
+    // Make sure SWG dialog background always intercept clicks to prevent users
+    // from interacting with the paywall page underneath.
+    const observer = new MutationObserver((mutationsList) => {
```

Comment:
nit: We don't need a reference to the variable. we could remove the variable and chain the `.observe`.